draft-ietf-tls-tls13.md

```diff
@@ -885,16 +885,17 @@ If the client has not provided a sufficient "key_share" extension (e.g., it
 includes only DHE or ECDHE groups unacceptable or unsupported by the
 server), the server corrects the mismatch with a HelloRetryRequest and
 the client needs to restart the handshake with an appropriate
-"key_share" extension, as shown in Figure 2. 
+"key_share" extension, as shown in Figure 2.
 If no common cryptographic parameters can be negotiated,
 the server MUST abort the handshake with an appropriate alert.
 
 ~~~
          Client                                               Server
 
          ClientHello
-           + key_share             -------->
+         + key_share               -------->
                                    <--------       HelloRetryRequest
+                                                         + key_share
 
          ClientHello
            + key_share             -------->
```

Comment:
It seems that indentation for this is rather inconsistent. If you're changing it above, I'd suggest making this and others match as well.